modules/__tests__/resolve-styles-test.js

```diff
@@ -395,10 +391,10 @@ describe('resolveStyles', function () {
         result.props[onHandlerName]();
         result.props[offHandlerName]();
 
-        expect(originalOffHandler).toBeCalled();
+        expect(originalOffHandler).to.have.been.called;
 
         result = resolveStyles(component, renderedElement);
-        expect(result.props.style.background).toEqual('blue');
+        expect(result.props.style.background).to.equal('blue');
```

Comment:
And again, for the future, would get much better assert failures here with something like:

```js
expect(result.prop).to.have.deep.property('style.background', 'blue');
```